data/gameSquares.json

```diff
@@ -1,146 +1,338 @@
+      "nextValidSquares": [ "b2", "b4", "d4", "d2" ],
+      "classes": "square black-square"
+    },
+    {
+      "id": "d3",
+      "nextValidSquares": null,
+      "classes": "square red-square"
     },
     {
       "id": "e3",
-      "nextValidSquares": [ "d2", "d4", "f4", "f2" ]
+      "nextValidSquares": [ "d2", "d4", "f4", "f2" ],
+      "classes": "square black-square"
+    },
+    {
+      "id": "f3",
+      "nextValidSquares": null,
+      "classes": "square red-square"
     },
     {
       "id": "g3",
-      "nextValidSquares": [ "f2", "f4", "h4", "h2" ]
+      "nextValidSquares": [ "f2", "f4", "h4", "h2" ],
+      "classes": "square black-square"
+    },
+    {
+      "id": "h3",
+      "nextValidSquares": null,
+      "classes": "square red-square"
     }
   ],
   "row2": [
+    {
+      "id": "a2",
+      "nextValidSquares": null,
+      "classes": "square red-square"
+    },
     {
       "id": "b2",
-      "nextValidSquares": [ "a1", "a3", "c3", "c1" ]
+      "nextValidSquares": [ "a1", "a3", "c3", "c1" ],
+      "classes": "square black-square"
+    },
+    {
+      "id": "c2",
+      "nextValidSquares": null,
+      "classes": "square red-square"
     },
     {
       "id": "d2",
-      "nextValidSquares": [ "c1", "c3", "e3", "e1" ]
+      "nextValidSquares": [ "c1", "c3", "e3", "e1" ],
+      "classes": "square black-square"
+    },
+    {
+      "id": "e2",
+      "nextValidSquares": null,
+      "classes": "square red-square"
     },
     {
       "id": "f2",
-      "nextValidSquares": [ "e1", "e3", "h3", "h1" ]
+      "nextValidSquares": [ "e1", "e3", "h3", "h1" ],
+      "classes": "square black-square"
+    },
+    {
+      "id": "g2",
+      "nextValidSquares": null,
+      "classes": "square red-square"
     },
     {
       "id": "h2",
-      "nextValidSquares": [ "g1", "g3" ]
+      "nextValidSquares": [ "g1", "g3" ],
+      "classes": "square black-square"
     }
   ],
   "row1": [
     {
       "id": "a1",
-      "nextValidSquares": [ "b2" ]
+      "nextValidSquares": [ "b2" ],
+      "classes": "square black-square"
+    },
+    {
+      "id": "b1",
+      "nextValidSquares": null,
+      "classes": "square red-square"
     },
     {
       "id": "c1",
-      "nextValidSquares": [ "b2", "d2" ]
+      "nextValidSquares": [ "b2", "d2" ],
+      "classes": "square black-square"
+    },
+    {
+      "id": "d1",
+      "nextValidSquares": null,
+      "classes": "square red-square"
     },
     {
       "id": "e1",
-      "nextValidSquares": [ "d2", "f2" ]
+      "nextValidSquares": [ "d2", "f2" ],
+      "classes": "square black-square"
+    },
+    {
+      "id": "f1",
+      "nextValidSquares": null,
+      "classes": "square red-square"
     },
     {
       "id": "g1",
-      "nextValidSquares": [ "f2", "h2" ]
+      "nextValidSquares": [ "f2", "h2" ],
+      "classes": "square black-square"
+    },
+    {
+      "id": "h1",
+      "nextValidSquares": null,
+      "classes": "square red-square"
```

Comment:
here I've added a `"classes"` property to each of the square objects so I can access the values easily in the `v-for` loop in `Board.vue`, and I've created individual objects for each of the red squares, though I've purposely given these `"nextValidSquares": null` so we can check them later with javascript for `!== null`.